### ISSUE TITLE
Idea: could we use the github machine learning good-first-issues lists?

### ISSUE THREAD

**Issue by Beanow** [rseng/good-first-issues#3]
https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source

Example: https://github.com/sourcecred/sourcecred/contribute

At first glance doesn't seem to have a clean API to use, but would be a nice improvement if we *could* use it. As it would make the result quality better while no longer needing to maintain a list of tags :smile:

**Comment by vsoch** [comment]
Yeah! I actually poked around hoping that there would be an api endpoint for that, but I didn't find anything so I opted for the manually done "good first issues" via the issues API. We definitely should look into adding this if GitHub ever exposes an endpoint! The action currently supports more than one org, so the current need isn't fully met, and it would really need to be an endpoint and not some hacky scraping thing :P 

Let's keep an eye out for this endpoint!

**Comment by vsoch** [comment]
I just pinged them about it on Twitter (by the way, do you have a Twitter handle? There have easily been 2-3 times I've wanted to "at" you but I couldn't find you!
